web/docs/advanced/deployment/manually.md

```diff
@@ -41,33 +41,36 @@ You'll have to [switch to PostgreSQL](/docs/data-model/backends#migrating-from-s
 
 There's a Dockerfile that defines an image for building the server in the `.wasp/build` directory.
 
-To run the server in production, deploy this Docker image to a hosting provider and ensure it has access to correct environment variables (this varies depending on the provider).
+To run the server in production, deploy this Docker image to a hosting provider and ensure required environment variables on the provider are correctly set up (mechanism of setting these up is specific per provider).
 All necessary environment variables are listed in the next section.
 
 #### Environment Variables
 
-Here are the environment variables your server requires to run:
+Here are the environment variables your server will be looking for:
 
-- `PORT`
-
-  The server's HTTP port number. This is where the server listens for requests (e.g., `3001`).
-
-- `DATABASE_URL`
+- `DATABASE_URL` (required)
 
   The URL of the Postgres database you want your app to use (e.g., `postgresql://mydbuser:mypass@localhost:5432/nameofmydb`).
 
-- `WASP_WEB_CLIENT_URL`
+- `WASP_WEB_CLIENT_URL` (required)
```

Comment:
We agreed on using the `<Required />` component

`import { Required } from '@site/src/components/Required'`

Reply:
Ok, I did the change, but just a note: this is not an API reference, instead this is listing env variables that server can expect.